### PR TITLE
NC 19 development

### PR DESCRIPTION
**Pull request by rullzer**
We branched of stable18. So time to bump the version number.

Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]
:tada:

**Review by ChristophWurst — APPROVED** [review]

**Review by georgehrke — APPROVED** [review]

File: version.php [hunk 1]
```diff
@@ -29,15 +29,15 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = array(18, 0, 0, 8);
+$OC_Version = array(19, 0, 0, 0);
 
 // The human readable string
-$OC_VersionString = '18.0.0 RC1';
+$OC_VersionString = '18.0.0 alpha';
```
Comment (gary-kim) [inline version.php]:
```suggestion
$OC_VersionString = '19.0.0 alpha';
```
Shouldn't this be 19 alpha?

Comment (rullzer) [reply version.php]: nah we always lower cased it...

Comment (skjnldsv) [reply version.php]: What about the 19 part :p

Comment (gary-kim) [reply version.php]: Oh okay, just got the impression from here
https://github.com/nextcloud/server/blob/04ef434af079e22ef27a874975970fde4ae93961/version.php#L35

Also, still need to bump the version number in the name.

Comment (rullzer) [reply version.php]: ah hehe fair enough but CI also fails. let me fix it all

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**